developerportal/context_processors.py

```diff
@@ -30,6 +35,29 @@ def directory_pages(request):
     }
 
 
+def topics_title(request):
+    """Get a name to use for the section formerly known as "Topics", based on
+    whatever the Topics page is currently called.
+
+    NB: this assumes the Topics page's title is a plural."""
+    from .apps.topics.models import Topics
+
+    _key = Topics.CACHE_KEY_TOPICS_TITLE
+
+    title = cache.get(_key)
+    if not title:
+        topics_page = Topics.published_objects.first()
```

Comment:
Wondering if dependening on the published order is not a little brittle. Could we maybe use something like the page id to retrieve the title? I guess right now we're only caching it for a single page, so maybe I'm jumping the gun.

Reply:
In a non-Wagtail world, I totally agree with you, but the structure of the site is such that there will (in real terms) only ever be one Topics page -- if there is more than one, the site will be effectively broken/in a weird state. As such, expecting just one `Topics` page to exist is not an unreasonable expectation.

Does that help ease your (excellent) conscience?

Reply:
Shall we then maybe use the `pk` defined in the fixtures instead of relying on the order of topics? That would feel better to me. I think I'm feeling better about making the general assumption about there only being one topics page. Thanks for the elaboration :)

Reply:
> Could we maybe use something like the page id to retrieve the title

Given the autoincrement state of the DB may be different across local dev or the dev|stage|prod k8s envs, `.first()` seems less brittle than going with IDs. And because the `/topics/` slug isn't guaranteed either, I think what we have is prob the best pick of the bunch

Reply:
Oh, so the pk in the fixtures
https://github.com/mdn/developer-portal/blob/34174f1313287d16f6d5f5fd7f79e853ec0f043c/developerportal/apps/common/fixtures/common.json#L652-L657
is not necessarily respected?

Huh tough nut indeed!

Reply:
Thinking about it: in tests and local setup, yes, it's used, but not in all the k8s deployments, where pages in Prod and Dev were created by hand. Stage may have used common.json tho

Reply:
I see, dang! Doesn't that then open even more room for them to be mis-ordered? Could we maybe check (once) which page is the one that contains all the topics and then use that one here?

Reply:
Given that we "trust" that there will only be one `Topics` page (which is a unique subclass of `BasePage` and the only page that can have `Topic` (singular) pages as children), I don't think we'll have a problem with the wrong `Topics` page being picked. 

However, your point does make me think that we can add a constraint to certain page types (ie, `Topics`, `Articles`, `Events`, `People`) to enforce the idea that we can only allow one single instance of these structural/grouping pages to exist. That would solve this risk of multiple `Topics` pages, by protecting us from actually adding more than one. Does that work for you?

Reply:
Oh, I think I was confused about that. There'll only ever be one instance of `Topics`? I guess everything I would ask as a follow-up would break the scope of this PR and would fall more under general database design curiosity. So for this thing here I'm in agreement, thanks for enlightening me :)